daemon/mgr/spec.go

```diff
@@ -12,11 +12,12 @@ import (
 type SpecWrapper struct {
 	s *specs.Spec
 
-	ctrMgr  ContainerMgr
-	volMgr  VolumeMgr
-	netMgr  NetworkMgr
-	prioArr []int
-	argsArr [][]string
+	ctrMgr     ContainerMgr
+	volMgr     VolumeMgr
+	netMgr     NetworkMgr
+	prioArr    []int
```

Comment:
Could you add more annotation to field `prioArr ` and `argsArr `?

Reply:
Not know prioArr and argsArr  actually mean, unrelated with systemd driver